packages/contracts-bedrock/deploy-config/final-migration-rehearsal.json

```diff
@@ -0,0 +1,39 @@
+{
+  "finalSystemOwner": "0x62790eFcB3a5f3A5D398F95B47930A9Addd83807",
+  "controller": "0x2d30335B0b807bBa1682C487BaAFD2Ad6da5D675",
+
+  "l1StartingBlockTag": "0x4104895a540d87127ff11eef0d51d8f63ce00a6fc211db751a45a4b3a61a9c83",
+  "l1ChainID": 5,
+  "l2ChainID": 420,
+  "l2BlockTime": 2,
+
+  "maxSequencerDrift": 1200,
+  "sequencerWindowSize": 3600,
+  "channelTimeout": 120,
+
+  "p2pSequencerAddress": "0xCBABF46d40982B4530c0EAc9889f6e44e17f0383",
+  "batchInboxAddress": "0xff00000000000000000000000000000000000420",
+  "batchSenderAddress": "0x3a2baA0160275024A50C1be1FC677375E7DB4Bd7",
+  "l2OutputOracleSubmissionInterval": 20,
+  "l2OutputOracleStartingTimestamp": 1670625264,
+  "l2OutputOracleStartingBlockNumber": 3324764,
+  "l2OutputOracleProposer": "0x88BCa4Af3d950625752867f826E073E337076581",
+  "l2OutputOracleChallenger": "0x88BCa4Af3d950625752867f826E073E337076581",
+  "finalizationPeriodSeconds": 2,
+
+  "proxyAdminOwner": "0x62790eFcB3a5f3A5D398F95B47930A9Addd83807",
+
+  "governanceTokenName": "Optimism",
+  "governanceTokenSymbol": "OP",
+  "governanceTokenOwner": "0x038a8825A3C3B0c08d52Cc76E5E361953Cf6Dc76",
+
+  "l2GenesisBlockGasLimit": "0x17D7840",
+  "l2GenesisBlockCoinbase": "0x4200000000000000000000000000000000000011",
```

Comment:
I opened a PR here to eliminate the need for this config option as it should always be the sequencer fee vault

https://github.com/ethereum-optimism/optimism/pull/4577

Reply:
Roger. Since we need this for tomorrow morning + this config is temporary I'm going to merge this. The real deploy config won't have the Coinbase field.